cmd/cli/main.go

```diff
@@ -28,18 +28,18 @@ func main() {
 	app.EnableBashCompletion = true
 	app.Flags = []cli.Flag{
 		cli.StringFlag{
-			EnvVar: "DRONE_TOKEN,WOODPECKER_TOKEN",
+			EnvVar: "WOODPECKER_TOKEN",
 			Name:   "t, token",
```

Comment:
Is this just a strange name or a way of writing alternatives like `EnvVar`?

Reply:
I would add a TODO but not start refactoring this here

Reply:
```suggestion
                        // TODO: rename to `token`
			Name:   "t, token",
```